packages/nc-gui-v2/pages/signup/[[token]].vue

```diff
@@ -152,6 +154,18 @@ function resetError() {
               </span>
             </button>
 
+            <a
+              v-if="appInfo.googleAuthEnabled"
+              href="http://localhost:8080/auth/google"
```

Comment:
hardcoded href? better add a TODO tag for further work.

Reply:
Yeah, I was going to do that 👍